daemon/call.c

```diff
@@ -113,37 +114,8 @@ const struct transport_protocol transport_protocols[] = {
 };
 const int num_transport_protocols = G_N_ELEMENTS(transport_protocols);
 
-static const char * const __term_reason_texts[] = {
-	[TIMEOUT] = "TIMEOUT",
-	[REGULAR] = "REGULAR",
-	[FORCED] = "FORCED",
-	[SILENT_TIMEOUT] = "SILENT_TIMEOUT",
-	[FINAL_TIMEOUT] = "FINAL_TIMEOUT",
-};
-static const char * const __tag_type_texts[] = {
-	[FROM_TAG] = "FROM_TAG",
-	[TO_TAG] = "TO_TAG",
-};
-static const char *const __opmode_texts[] = {
-	[OP_OFFER] = "offer",
-	[OP_ANSWER] = "answer",
-};
-
-static const char * get_term_reason_text(enum termination_reason t) {
-	return get_enum_array_text(__term_reason_texts, t, "UNKNOWN");
-}
-const char * get_tag_type_text(enum tag_type t) {
```

Comment:
Can you also please move the relevant header definitions from `call.h` to `cdr.h`